optashift-employee-rostering-client/src/main/java/org/optaplanner/openshift/employeerostering/client/ServiceClientFactory.java

```diff
@@ -0,0 +1,67 @@
+package org.optaplanner.openshift.employeerostering.client;
+
+import java.net.URL;
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.client.jaxrs.ResteasyWebTarget;
+import org.optaplanner.openshift.employeerostering.shared.employee.EmployeeRestService;
+import org.optaplanner.openshift.employeerostering.shared.roster.RosterRestService;
+import org.optaplanner.openshift.employeerostering.shared.shift.ShiftRestService;
+import org.optaplanner.openshift.employeerostering.shared.skill.SkillRestService;
+import org.optaplanner.openshift.employeerostering.shared.spot.SpotRestService;
+import org.optaplanner.openshift.employeerostering.shared.tenant.TenantRestService;
+
+/**
+ * Creates client proxies to remote REST API endpoints.
+ * */
+public final class ServiceClientFactory {
+
+    private static final String REST_API_CONTEXT = "rest";
+
+    private URL baseUrl;
+    private ResteasyClient resteasyClient;
+    private ResteasyWebTarget target;
+
+    /**
+     * Constructor.
```

Comment:
Let's remove this line. Javadocs should only include info that the method signature doesn't already say.